camunda-transaction-boundaries-plugin/README.md

```diff
@@ -1,9 +1,18 @@
 # Camunda Transaction Boundaries Plugin
 
-[![Compatible with Camunda Modeler version 2.2](https://img.shields.io/badge/Camunda%20Modeler-2.2+-blue.svg)](https://github.com/camunda/camunda-modeler)
+[![Compatible with Camunda Modeler version 5](https://img.shields.io/badge/Modeler_Version-5.0.0+-blue.svg)](#) [![Plugin Type](https://img.shields.io/badge/Plugin_Type-BPMN_(Camunda_Platform_7)-orange.svg)](#)
+
+## Functionality
 
 [Transaction Boundaries](https://github.com/bpmn-io/camunda-transaction-boundaries/) as a plugin for the Camunda Modeler that visualizes the (database) transaction boundaries Camunda applies.
 
 Put this directory into the `plugins` directory of the Camunda Modeler and you're ready to go.
 
-If you're interested in how to create your own plugins see the [documentation](https://github.com/camunda/camunda-modeler/tree/master/docs/plugins) and this [example](https://github.com/camunda/camunda-modeler-plugin-example).
+If you're interested in how to create your own plugins see the [documentation](https://docs.camunda.io/docs/components/modeler/desktop-modeler/plugins/) and this [example](https://github.com/camunda/camunda-modeler-plugin-example).
+
+## Compatibility Notice
+
+| Plugin Version | Compatible With Camunda Modeler |
+|-|-|
+| <= 0.4.2 | 2.2 - 4.x |
+| >= 0.5.0 | 5.0.0+ |
```

Comment:
How do we release this plugin? Just updating the version in package.json? If so, would we not need to do this as part of this PR?

Reply:
That's a good point, I'll double check but I think you are right.